pkg/trimaran/loadvariationriskbalancing/analysis.go

```diff
@@ -105,8 +106,8 @@ func createResourceStats(metrics []watcher.Metric, node *v1.Node, podRequest *fr
 	rs.usedAvg = nodeUtil * rs.capacity / 100
 	rs.usedStdev = nodeStd * rs.capacity / 100
 
-	klog.V(6).Infof("resource %s usage statistics for node %s: capacity=%f; req=%f; usedAvg=%f; usedStdev=%f",
-		watcherType, node.GetName(), rs.capacity, rs.req, rs.usedAvg, rs.usedStdev)
+	klog.V(6).InfoS("resource usage statistics for node:", "node", klog.KObj(node), " capacity", rs.capacity, 
```

Comment:
Capitalize the first word, and remove the ending `:`, as well as empty spaces. (applies elsewhere)